src/StoreForwarding.cpp

```diff
@@ -1222,17 +1223,17 @@ class RenameVars : public IRMutator {
     }
 };
 
-vector<Store> find_non_aliasing_stores(Stmt stmt) {
+vector<const Store *> find_non_aliasing_stores(Stmt stmt) {
```

Comment:
No, it makes sense that the copy constructor shouldn't exist. I think the right thing to do would be to change the vector to hold Stmt instead of Store. Then, where you consume this, you would need to do (iter)->as<Store>() to get a pointer to the Store.